examples/09_shop_content/02_get_shop_policies_by_product_handle/query.graphql

```diff
@@ -0,0 +1,32 @@
+
+# A store may wish to show store policies on the product page, as the poilcies could affect a buyer's decision to purchase from one shop over another.
+
+# This query adds a Shop Policy query to a product query and returns the shop policies.
+
+query getShopProductbyProductHandle($productHandle: String!) {
+  product(handle: $productHandle) {
+    id
+  }
+  shop {
+    privacyPolicy {
+      body # Policy text, maximum size of 64kb.
+      title
+    }
+    refundPolicy {
+      body
+      title
+    }
+    shippingPolicy {
+      body
+      title
+    }
+    termsOfService {
+      body
+      title
+    }
+    subscriptionPolicy {
+      body
+      title
+    }
+  }
+}
```

Comment:
I'm wondering if this would be useful as it demonstrates a combined query and not necessarily a connection between products and policies.

Reply:
Good point. It's a example taken from marketplace docs and is kind of out scope for the starter kit imho anyways